src/api/operations/client.fragments.graphql

```diff
@@ -33,6 +33,13 @@ fragment ClientName on Client {
   nameSuffix
 }
 
+fragment ClientPermissions on Client {
```

Comment:
Mind if I remove this, since it's not used? I reorganized the fragments to hopefully make things clearer, and part of that was moving all the access fragments into one file and setting them up ON the access objects, not on the parent client.